Build fails for me, do you maybe know what could be the issue?

```
> PREFIX=build make install
cc -O3 -Wall -Wextra -Isrc -std=gnu99 -fPIC   -c -o src/parser.o src/parser.c
In file included from src/parser.c:1:
src/parser.c: In function ‘ts_lex_keywords’:
src/tree_sitter/parser.h:136:8: warning: variable ‘eof’ set but not used [-Wunused-but-set-variable]
  136 |   bool eof = false;             \
      |        ^~~
src/parser.c:17793:3: note: in expansion of macro ‘START_LEXER’
17793 |   START_LEXER();
      |   ^~~~~~~~~~~
ar rcs libtree-sitter-purescript.a src/parser.o src/scanner.o
cc  -shared -Wl,-soname,libtree-sitter-purescript.so.0 src/parser.o src/scanner.o  -o libtree-sitter-purescript.so.0.0
ln -sf libtree-sitter-purescript.so.0.0 libtree-sitter-purescript.so
ln -sf libtree-sitter-purescript.so.0.0 libtree-sitter-purescript.so.0
install -d 'build/lib'
install -m755 libtree-sitter-purescript.a 'build/lib'/libtree-sitter-purescript.a
install -m755 libtree-sitter-purescript.so.0.0 'build/lib'/libtree-sitter-purescript.so.0.0
ln -sf libtree-sitter-purescript.so.0.0 'build/lib'/libtree-sitter-purescript.so.0
ln -sf libtree-sitter-purescript.so.0.0 'build/lib'/libtree-sitter-purescript.so
install -d 'build/include'/tree_sitter
install -m644 bindings/c/purescript.h 'build/include'/tree_sitter/
install: cannot stat 'bindings/c/purescript.h': No such file or directory
make: *** [Makefile:88: install] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -85,15 +85,6 @@ install: all
 	ln -sf libtree-sitter-$(PARSER_NAME).$(SOEXTVER) '$(DESTDIR)$(LIBDIR)'/libtree-sitter-$(PARSER_NAME).$(SOEXTVER_MAJOR)
 	ln -sf libtree-sitter-$(PARSER_NAME).$(SOEXTVER) '$(DESTDIR)$(LIBDIR)'/libtree-sitter-$(PARSER_NAME).$(SOEXT)
 	install -d '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter
-	install -m644 bindings/c/$(PARSER_NAME).h '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter/
-	install -d '$(DESTDIR)$(PCLIBDIR)'
-	sed -e 's|@LIBDIR@|$(LIBDIR)|;s|@INCLUDEDIR@|$(INCLUDEDIR)|;s|@VERSION@|$(VERSION)|' \
-	    -e 's|=$(PREFIX)|=$${prefix}|' \
-	    -e 's|@PREFIX@|$(PREFIX)|' \
-	    -e 's|@ADDITIONALLIBS@|$(ADDITIONALLIBS)|' \
-		-e 's|@PARSERNAME@|$(PARSER_NAME)|' \
-		-e 's|@PARSERREPOURL@|$(PARSER_REPO_URL)|' \
-	    bindings/c/tree-sitter.pc.in > '$(DESTDIR)$(PCLIBDIR)'/tree-sitter-$(PARSER_NAME).pc
 
 clean:
 	rm -f $(OBJ) libtree-sitter-$(PARSER_NAME).a libtree-sitter-$(PARSER_NAME).$(SOEXT) libtree-sitter-$(PARSER_NAME).$(SOEXTVER_MAJOR) libtree-sitter-$(PARSER_NAME).$(SOEXTVER) bindings/c/$(PARSER_NAME).h

